### DISCUSSION TITLE
Cable Trace Fiber not working as expected

### DISCUSSION THREAD

**Discussion by nbrgit** [netbox-community/netbox#10998] [General]
Hey All,
some Question, I try to Trace a Rout but it dosn't work.

I've connect an Interface from a Switch To a Fiber Patchpanel The Ports are Splitted like 1A 1B, 2A 2B, ...
In my Opinion it's the right way - because i need a Port for each Fiber.

If i make a trace from the Interface it Only shows the First Connection Interface -> 1A 1B and then "Path Splited" so the Route isn't displayed completely.

Only If I Connect one fiber the Trace is completely.

Any Idea?

**Comment by arleinalesso** [comment]
I'm having the same problem.

![image](https://user-images.githubusercontent.com/2326549/209166751-78bfd8af-7373-4481-ba31-f3feccf688c8.png)


Before click link

![image](https://user-images.githubusercontent.com/2326549/209166962-a2c01c2e-3951-47cd-888e-d061def3565a.png)

**Comment by mmfreitas** [comment]
I believe this is intended, check this https://github.com/netbox-community/netbox/issues/11184#issuecomment-1355636219.
